service/reducers/entities/users.js

```diff
@@ -174,6 +174,15 @@ function statuses(state = {}, action) {
     }
 }
 
+function autocompleteUsersInChannel(state = {}, action) {
```

Comment:
the thing @hmhealey is that it actually is a client method, and this can be use when we port redux to the webapp, so I think we should leave it here